t2.lic

```diff
@@ -28,6 +33,10 @@ class T2
         fput("avoid #{avoid['type']}")
       end
     end
+
+    unless (Time.now - DRSkill.start_time > @settings.t2_startup_delay.to_i) || @args.nodelay
```

Comment:
💡 This conditional could probably be simplified to something like `if !@args.nodelay && @settings.t2_startup_delay.to_i > 0` since, worst case, it's already been long enough and the next line gets skipped by _it's_ conditional.

I find this version more readable, but it's not a big deal.

Reply:
Thank you. I always appreciate your time, and help.